server/models/Activity.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(Sequelize, DataTypes) {
     updatedAt: false,
 
     hooks: {
-      afterCreate: function(activity) {
+      afterCreate(activity) {
```

Comment:
Did you test this? Would have thought you would need to do `afterCreate: activity => {`